index.html

```diff
@@ -18,7 +18,7 @@
   <link rel="shortcut icon" href="img/favicon.png">
   <link rel="stylesheet" href="css/simple-grid.css">
   <link rel="stylesheet" href="css/main.css">
-  <title>Sunburst Smorgasbord | a tool to describe your relationships.</title>
```

Comment:
I would still want to keep the reference to the Smorgasbord preferably as it is a clear reference to an already known thing in relationship anarchy; also if we do decide to remove it we also need to figure out a new subdomain